var/spack/repos/builtin/packages/flecsi/package.py

```diff
@@ -69,18 +69,25 @@ class Flecsi(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("boost@1.79.0:", when="@2.2:")
     depends_on("kokkos@3.2.00:", when="+kokkos")
     depends_on("kokkos@3.7:", when="+kokkos @2.3:")
+    depends_on("kokkos@3.7:", when="@2.4:")
     depends_on("kokkos +cuda +cuda_constexpr +cuda_lambda", when="+kokkos +cuda")
```

Comment:
```suggestion
    depends_on("kokkos +cuda", when="+kokkos +cuda")
```
to leverage the below, or should it be
```py
    requires("^kokkos +cuda", when="+kokkos +cuda")
```
?